### ISSUE TITLE
Mouse cursor selection, drag and drop based copy pasting of values, as in buzz [FR]

### ISSUE THREAD

**Issue by michaelsjackson** [JoepVanlier/Hackey-Trackey#36]
Mouse cursor, drag and drop, copy paste as available in buzz is not possible, yet.

**Comment by michaelsjackson** [comment]
It could be this is impossible, I am not sure if I know this from buzz itself or its variant "buze".

**Comment by JoepVanlier** [comment]
What do you mean by mouse cursor drag and drop? That you select a block, then drag it to a new area?

**Comment by michaelsjackson** [comment]
Yes you select a range of cells, then drag and drop anywhere else, and those copies overwrite what was there previously. So its a quick form of copy paste, using the mouse. Usually, if I like a few sounds in the pattern, I can select them, and place them in a few more interesting places. Because it sounded good.

**Comment by michaelsjackson** [comment]
If its a big task, maybe not a must, as long as there other useful ideas, to add and modify. But good to know, this is possible in theory. Nothing hurries anyway. You can check buze at some point of you want, just to see if it feels cool and useful enough, from your perspective.

I would suggest an older buze version for testing, 0.5.9, as only this version is like real copy of original buzz, later version or the current 0.8.9 they changed too many things, removing the simplicity of buzz. For original buzz feeling buze 0.5.9 is best.
